docs/Getting Started/OpenSUSE/index.rst

```diff
@@ -0,0 +1,104 @@
+.. highlight:: sh
+
+openSUSE
+========
+
+.. contents:: Table of Contents
+  :local:
+
+Installation
+------------
+
+If you want to use ZFS as your root filesystem, see the `Root on ZFS`_
+links below instead.
+
+ZFS packages are not included in official openSUSE repositories, but repository of `filesystems projects of openSUSE
+<https://software.opensuse.org/download.html?project=filesystems&package=zfs>`__ 
+includes such packages of filesystems including OpenZFS.
+
+openSUSE progresses through 3 main distribution branches, these are called Tumbleweed, Leap and SLE. There are ZFS packages available for all three.
+
+
+Command Line Installation
+-------------------------
+
+#. For openSUSE Tumbleweed::
```

Comment:
Would it make sense to remove some of the duplication? For example, maybe move the `zypper refresh` and `zypper install` to be a second step? Or maybe have a list/table of URLs or something? I don't know if this would be better or worse; it's just a thought. Also, are all of these still supported by openSUSE (in terms of security updates)? If not, there's no need for us to list them.

Reply:
I'm looking for it right for now.